openshift/e2e-common.sh

```diff
@@ -6,6 +6,35 @@ export ZIPKIN_NAMESPACE=$EVENTING_NAMESPACE
 export KNATIVE_DEFAULT_NAMESPACE=$EVENTING_NAMESPACE
 export CONFIG_TRACING_CONFIG="test/config/config-tracing.yaml"
 
+if [ -n "$OPENSHIFT_BUILD_NAMESPACE" ]; then
```

Comment:
@mgencur  should we do something like here: 
https://github.com/openshift/knative-eventing/pull/1394/files

as well ?

Reply:
Yeah. I think you're correct.